qemu/tests/cfg/ovmf_check_efi.cfg

```diff
@@ -0,0 +1,11 @@
+- ovmf_check_efi:
+    only q35
+    only ovmf
+    type = ovmf_check_efi
+    Linux:
+        gdisk_cmd = "gdisk -l /dev/[vs]da"
+        efi_info = "EFI System Partition"
+        dmesg_cmd = "dmesg | grep EFI"
+    Windows:
+        bios_cmd = "systeminfo | findstr BIOS"
+        bios_ver = "EFI Development Kit"
```

Comment:
Not sure if you can use same variable names for `gdisk_cmd` and`bios_cmd`,  `efi_info` and `bios_ver`, so in the test script, it could reduce code redundant.

Reply:
Agree with @hereischen , they are actually same action, which can have the detail commands and check string as their variants.